plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -433,3 +433,16 @@ async function safeExecuteInTheMiddleAsync<T>(
     return result as T;
   }
 }
+
+export const createResolveSpanName = (fieldName: string): string => {
+  return `${SpanNames.RESOLVE}.${fieldName}`;
+};
+
+export const createExecuteSpanName = (
+  operationType: string | undefined
+): string => {
+  if (operationType) {
+    return `${SpanNames.EXECUTE}.${operationType}`;
+  }
+  return `${SpanNames.EXECUTE}`;
+};
```

Comment:
What's your motivation in adding high-cardinality (field name) and unbounded-cardinality (operation name) data to these spans' names? I'm concerned that this change will make queries on these span names harder, since these names would now have two jobs: convey the span "type" + convey another aspect of the span. E.g., it's harder to refer to all the "resolver" spans. The added data is present in other attributes; is that not sufficient? Each attribute describing a single characteristic of the span? If one wished to do a query akin to  how "GROUP BY name" would be under these changes, one would do "GROUP BY name, field_name".

Reply:
Apologies; just saw the linked issue. Commented there.

Reply:
I believe it's adding more visibility (without looking at attributes) at what part of "graphql resolving" does the span belongs to. Same as for HTTP there isn't just "http.resolving" but it includes method (GET, POST etc.) and also the "route" such as `/api/users/{user_id}`.

https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/http/#name